cycler.py

```diff
@@ -329,10 +329,55 @@ def simplify(self):
         # I would believe that there is some performance implications
 
         trans = self._transpose()
-        return reduce(add, (cycler(k, v) for k, v in six.iteritems(trans)))
+        return cycler(**dict((k, v) for k, v in six.iteritems(trans)))
 
 
-def cycler(label, itr):
+def cycler(*args, **kwargs):
+    """
+    Create a new `Cycler` object from the combination of
+    positional arguments or keyword arguments.
+
+    cycler(arg)
+    cycler(label1=itr1[, label2=iter2[, ...]])
+
+    Form 1 simply copies a given `Cycler` object.
+    Form 2 composes a `Cycler` as an inner product of the
```

Comment:
"inner product" might not mean much to many docstring readers here. Maybe add something like, "All iterables are cycled simultaneously."